pwndbg/commands/version.py

```diff
@@ -10,6 +10,10 @@
 from __future__ import unicode_literals
 
 import sys
+import os
+from subprocess import check_output
+from platform import platform
+import argparse
```

Comment:
you need to properly reorder the imports to make `isort` happy, just install it and call it to see what the problem is :)